src/main/java/com/madgag/gif/fmsware/AnimatedGifEncoder.java

```diff
@@ -345,6 +372,30 @@ protected int findClosest(Color c) {
 	}
 	
 	/**
+	 * Returns index of palette color closest to c
+	 *
+	 */
+	protected int findExact (Color c) {
+		if (colorTab == null) return -1;
+		int r = c.getRed() & 0xff;
+		int g = c.getGreen() & 0xff;
+		int b = c.getBlue() & 0xff;

```

Comment:
Well, as I was not sure that the range of c.getXXX is the same, I tried to get the same range just by applying the same AND bitmask, but you're right it is not clearly needed.